src/get-started/install.md

```diff
@@ -84,6 +85,16 @@ You may also run the extended test suite located in the `sandbox` directory:
 cargo test --manifest-path sandbox/Cargo.toml
 ```
 
+Since version `0.2.0` Exonum supports RocksDB as alternative data storage.
+To enable RocksDB support you need to pass  additional parameter to cargo:
```

Comment:
"...as alternative data storage. To enable RocksDB support..." --> "...as an alternative data storage, to enable RocksDB support..."

Reply:
Are you sure? Seems like these are 2 separate sentences.

Reply:
@slowli ok, let it be 
"+As of version `0.2.0` Exonum supports RocksDB as alternative data storage.
+To enable RocksDB support you need to pass  additional parameter to cargo:"